renv/profiles/dev/renv.lock

```diff
@@ -424,8 +470,13 @@
     "noctua": {
       "Package": "noctua",
       "Version": "2.6.2",
-      "Source": "Repository",
-      "Repository": "CRAN",
+      "Source": "GitHub",
+      "RemoteType": "github",
+      "RemoteHost": "api.github.com",
+      "RemoteUsername": "DyfanJones",
+      "RemoteRepo": "noctua",
+      "RemoteRef": "master",
+      "RemoteSha": "23a4cfbf537407c7a1547fc13ba771ba2eb098e0",
```

Comment:
`noctua` didn't actually get bumped (in #62) to the version with the `unload` fix, so this PR does that as well as bumping/adding a few other dependencies.